docs/application/native/api/iot-headed/5.5/group__Evas__Font__Path__Group.html

```diff
@@ -124,7 +124,7 @@
   </table>
   </dd>
 </dl>
-<dl class="since"><dt><b>Since:</b></dt><dd>1.9 </dd></dl>
+<dl class="since"><dt><b>Since:</b></dt><dd>1.9  3.0f(2.3.1) </dd></dl>
```

Comment:
Would you please check the updates in this page?

Reply:
checked

Reply:
https://review.tizen.org/gerrit/#/c/platform/upstream/efl/+/250985/